app/src/test/java/com/x8bit/bitwarden/data/autofill/processor/AutofillProcessorTest.kt

```diff
@@ -82,13 +82,97 @@ class AutofillProcessorTest {
 
     @AfterEach
     fun teardown() {
-        verify(exactly = 1) {
-            dispatcherManager.unconfined
-        }
         unmockkStatic(::createAutofillSavedItemIntentSender)
         unmockkStatic(AutofillRequest.Fillable::toAutofillSaveItem)
     }
 
+    @Suppress("MaxLineLength")
+    @Test
+    fun `processFillRequest should allow additional requests to be invoked after cancellation signal is triggered`() {
```

Comment:
Question about ordering...would it make sense to make this come after some of the other tests that cover more of the standard logic first?